tests/unit/lms/services/mailchimp_test.py

```diff
@@ -0,0 +1,254 @@
+                                "from_email": sentinel.digests_from_email,
+                                "from_name": sentinel.digests_from_name,
+                                "to": [
+                                    {
+                                        "email": sentinel.email_1,
+                                        "name": sentinel.display_name_1,
+                                    }
+                                ],
+                                "track_opens": True,
+                                "track_clicks": True,
+                                "global_merge_vars": [
+                                    {"name": "num_annotations", "content": 2},
+                                    {
+                                        "name": "courses",
+                                        "content": [
+                                            {
+                                                "title": course.lms_name,
+                                                "num_annotations": 2,
+                                            }
+                                        ],
+                                    },
+                                ],
+                            },
+                            "async": True,
+                        }
+                    ),
+                    call(
+                        {
+                            "template_name": "instructor_digest",
+                            "message": {
+                                "subaccount": sentinel.digests_subaccount,
+                                "from_email": sentinel.digests_from_email,
+                                "from_name": sentinel.digests_from_name,
+                                "to": [
+                                    {
+                                        "email": sentinel.email_2,
+                                        "name": sentinel.display_name_2,
+                                    }
+                                ],
+                                "track_opens": True,
+                                "track_clicks": True,
+                                "global_merge_vars": [
+                                    {"name": "num_annotations", "content": 2},
+                                    {
+                                        "name": "courses",
+                                        "content": [
+                                            {
+                                                "title": course.lms_name,
+                                                "num_annotations": 2,
+                                            }
+                                        ],
+                                    },
+                                ],
+                            },
+                            "async": True,
+                        }
+                    ),
+                ]
+            ).only()
+        )
```

Comment:
Oh ... dear.

This should probably mock `MailchimpService.send_template()` and assert about how that is called rather than asserting about the much more complicated calls that're made to the mailchimp client itself.

But the other problem is the complexity of constructing a fake `digests` object (as would be returned by `DigestService`) to pass in to the method.